pom.xml

```diff
@@ -33,7 +33,7 @@
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-api</artifactId>
-    <version>2.5-SNAPSHOT</version>
+    <version>2.5-optimized3-SNAPSHOT</version>
```

Comment:
Then put a `<!-- FIXME remove custom version before merge-->` still a :bug: without the comment